front_end/scheduled_scripts/separate.py

```diff
@@ -0,0 +1,52 @@
+import os
+
```

Comment:
Please explain what this script is for.

Reply:
That's the script I was using to separate webserver and it slipped in, sorry! I removed it.